.travis.yml

```diff
@@ -1,48 +1,4 @@
-language: php
+version: ~> 1.0
 
-dist: xenial
-
-services:
-  - mysql
-  - postgresql
-
-cache:
-  directories:
-    - $HOME/.composer/cache/files
-
-matrix:
-  include:
-    - php: 5.6
-      env: DB=MYSQL RECIPE_VERSION=4.4.x-dev PHPUNIT_TEST=1 PHPCS_TEST=1
-    - php: 7.1
-      env: DB=MYSQL RECIPE_VERSION=4.5.x-dev PHPUNIT_COVERAGE_TEST=1 PDO=1
-    - php: 7.2
-      env: DB=PGSQL RECIPE_VERSION=4.6.x-dev PHPUNIT_TEST=1
-    - php: 7.3
-      env: DB=MYSQL RECIPE_VERSION=4.6.x-dev PHPUNIT_TEST=1 
-    - php: 7.4
-      env: DB=MYSQL RECIPE_VERSION=4.x-dev PHPUNIT_TEST=1
-
-before_script:
-  # Init PHP
-  - phpenv rehash
-  - phpenv config-rm xdebug.ini
-  - echo 'memory_limit = 2048M' >> ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/travis.ini
-
-  # Install composer dependencies
-  - composer validate
-  - composer require --no-update silverstripe/recipe-cms:$RECIPE_VERSION
-  # This module has integration testing requirements
-  - composer require --no-update silverstripe/subsites:^2 silverstripe/taxonomy:^2 silverstripe/contentreview:^4
```

Comment:
Weird... how does it work without these dependencies now?

Reply:
The following has been added:
```
env:
  global:
    - REQUIRE_EXTRA="silverstripe/subsites:^2 silverstripe/taxonomy:^2 silverstripe/contentreview:^4"
```